subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -173,8 +173,8 @@ config BT_PERIPHERAL_PREF_MAX_INT
 	help
 	  Range 3200 to 65534 is invalid. 65535 represents no specific value.
 
-config BT_PERIPHERAL_PREF_SLAVE_LATENCY
-	int "Peripheral preferred slave latency in Connection Intervals"
+config BT_PERIPHERAL_PREF_PERIPHERAL_LATENCY
+	int "Peripheral preferred peripheral latency in Connection Intervals"
```

Comment:
I think the initial "peripheral" is enough to indicate that this is in fact a peripheral option, and we can just call it `BT_PERIPHERAL_PREF_LATENCY`

Reply:
@joerchan raised the same point internally. I disagree because I think users will prefer to be able to match the full name "preferred peripheral latency" in the spec.

Reply:
@carlescufi What naming in the spec are you referring to here? 

The one I can find that's relative to this, is the "Peripheral Preferred Connection Parameters" which, in my opinion, matches `BT_PERIPHERAL_PREF_LATENCY` better than `BT_PERIPHERAL_PREF_PERIPHERAL_LATENCY`.

If you have some other value in mind from the spec, could you refer me to that?

Reply:
Hmm you are right actually. I will change this one.

Reply:
Oh I see now where I got it from. Turns out that the spec commonly uses  the term "Peripheral latency". But I still think your proposed change is fine.